docs/refarch/commonTermsConventions.md

```diff
@@ -0,0 +1,22 @@
+# Glossary of Common Terms and Conventions Used
+
+## Overview
+The following common terms and conventions are use in all of the Mojaloop Switch's Bounded Contexts and Use Cases.
+
+## Glossary
+
+|    **Convention/Term**    |       **Description**       |
+| :------------------------ | :-------------------------- |
+| Actors                                                            | Human or external system Use Case participant. All Use Cases are driven by Actors.|
+| BC                                                                | Bounded Context: A bounded-context is a component of Design-Driven Development and typically contains a single, or multiple subdomains. Bounded Contexts are Solution Space entities, and contain a single solution applicable to a single, or multiple, subdomain/s.<br/><br/>(*For further information please see: Bounded Contexts in the Solution Space overview, or References and Further Reading*)|
+| UC | Use Case: A Use Case is a list of actions or steps describing interactions between an Actor (human or external system) and a system to achieve a particular objective. An example of a Use Case from Mojaloop is "Perform Transfer with Payee Confirmation".<br/><br/>(*Further reading: "Use Case" (Definition) - From Wikipedia, the free encyclopedia*)|
+| Sync                                                              | Synchronous communications, one or two way, part of the originating process. Signified by a solid line in the UC schematics. Typically used for Messages that need to be included in a particular UC workflow in order for it to execute successfully. An example of synch Messaging is a query from the Transfers BC to the Participant Lifecycle Management BC to get Participant data if not cached to complete a Transfer request.|
+| Async                                                             | Asynchronous communications, one or two way, not part of the originating process. Signified by a dotted line in the UC schematics. Typically used for Events which are used to indicate something that has happened, and is immutable and won't change, for example callback reports.|
+| POST                                                              | Utilized to create new resources. In particular, it's used to create subordinate resources. That is, subordinate to some other (e.g. parent) resource. IOW, when creating a new resource, POST to the parent and the service takes care of associating the new resource with the parent, assigning an ID (new resource URI), etc. On successful creation, the system will return a Location header with a link to the newly-created resource with the 201 HTTP status.<br/><br/>(*Source: Restful API Tutor - see References and further reading*)|
```

Comment:
```suggestion
| POST                                                              | Utilized to create new resources. In particular, it's used to create subordinate resources. That is subordinate to some other (e.g. parent) resource. IOW, when creating a new resource, POST to the parent and the service takes care of associating the new resource with the parent, assigning an ID (new resource URI), etc. On successful creation, the system will return a Location header with a link to the newly-created resource with the 201 HTTP status.<br/><br/>(*Source: Restful API Tutor - see References and further reading*)|
```

Two things here:

1. `IOW` <-- please define
2. `(*Source: Restful API Tutor - see References and further reading*)` <-- this applies to the entire table, but why are we not using the `[^1]` plugin to handle these?

Reply:
1. IOW = In other words   -  I can update and repost.  Please advise if that is required, thanks.
2. The entire document needs a Further Reading list and/or bibliography.  I would like to do this in much the same way as I have for the Common Terms and Conventions Used.  I need to chat to you about this.  We can chat either today or tomorrow.

Thanks. -M.